Stellar-operation-payment.x

```diff
@@ -61,35 +71,59 @@ struct PaymentOp
 enum PaymentResultCode
 {
     // codes considered as "success" for the operation
+    //: Payment was successfully completed
     SUCCESS = 0, // payment successfully completed
 
     // codes considered as "failure" for the operation
-    MALFORMED = -1, // bad input
-    UNDERFUNDED = -2, // not enough funds in source account
-    LINE_FULL = -3, // destination would go above their limit
-	DESTINATION_BALANCE_NOT_FOUND = -4,
+    //: Malformed operation (e.g. payment sender balance ID and payment receiver balance ID are equal)
+    MALFORMED = -1,
+    //: Not enough funds in source account
+    UNDERFUNDED = -2,
+    //: After the payment a destination balance would go above the limit
+    LINE_FULL = -3,
+    //: There is no such destination balance
+    DESTINATION_BALANCE_NOT_FOUND = -4,
+    //: Sender balance asset and receiver balance asset are not equal
     BALANCE_ASSETS_MISMATCHED = -5,
-	SRC_BALANCE_NOT_FOUND = -6, // source balance not found
+    //: There is no such source balance
```

Comment:
Umm, actually. 
There is no balance found with ID provided in sourceBalanceID. For example